src/audio/general.c

```diff
@@ -4705,12 +4729,16 @@ s32 Audio_IsSequencePlaying(u16 seqId) {
 void func_800F5ACC(u16 seqId) {
     u16 curSeqId = Audio_GetActiveSeqId(SEQ_PLAYER_BGM_MAIN);
 
+#if !OOT_DEBUG
+    if (1) {}
+#endif
```

Comment:
does this work inside the if?

Reply:
It does not, because both `if (0) {}` and `if (1) {}` will mess up the debug version